routes/adminInvitation.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express')
+
+const langMiddleware = require('~/middlewares/language')
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const adminInvitationController = require('~/controllers/adminInvitation')
+
+const router = express.Router()
+
+router.post('/', langMiddleware, asyncWrapper(adminInvitationController.inviteAdmins))
```

Comment:
you have getAdminInvitations, prolly it would be better to call this one sendAdminInvitation?

Reply:
ok